mod_ci/controllers.py

```diff
@@ -517,7 +517,11 @@ def inform_mailing_list(mailer, id, title, author, body):
     mailer.send_simple_message({
         "to": "ccextractor-dev@googlegroups.com",
         "subject": subject,
-        "text": "{title} - {author}\n {body}".format(title=title, author=author, body=body)
+        "text": """{title} - {author}\n
+        Link to Issue: https://github.com/CCExtractor/ccextractor/issues/{issue_number}\n
```

Comment:
Please don't hard-code the link to the GitHub repository. We have some veriables for this which you can use, just search the code ;)